.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           name: check MyPy
           command: |
             . venv/bin/activate
-            mypy --ignore-missing-imports --check-untyped-defs --no-strict-optional src/
```

Comment:
Why did we do this?

Reply:
And remove evry installation in YAML files. That is not needed, and we don't intend to test EinsteinPy with latest sympy

Reply:
mypy was asking for type definition. I think we can revert this now

Reply:
Better revert it